hack/test-net-e2e.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
We already have a script (test/extended/networking.sh) that manages network test execution, including cluster deployment.  This script is intended to allow tests to be run against an arbitrary deployment that is not controlled by the test script, so that an arbitrary cluster can be targeted as part of development.  If you insist that test/extended is where it belongs, fair enough, but I don't think it should be 'follow the patterns there' since it's purpose is different from what those patterns intend.  Given that, what would you suggest?